src/data/store.test.jsx

```diff
@@ -0,0 +1,14 @@
+import { store } from './store';
+
+let courseContext;
+
+describe('API client requestAssets', () => {
+  it('throws error when courseDetails is undefined', () => {
+    expect(store.rootReducer.courseDetails.toThrow('Error: Missing Course Context'));
+  });
+
+  it('sets courseDetails properly when defined', () => {
+    courseContext = {};
+    expect(store.rootReducer.courseDetails).toEqual(courseContext);
+  });
+});
```

Comment:
I'm able to get to the courseDetails reducer, but it's giving the following error and the `let courseContext` above doesn't work :(

 `FAIL  src/data/store.test.jsx
  ● Test suite failed to run

    ReferenceError: courseContext is not defined
      
      at courseDetails (src/data/reducers/index.js:8:3)
      at node_modules/redux/lib/combineReducers.js:53:24
          at Array.forEach (<anonymous>)
      at assertReducerShape (node_modules/redux/lib/combineReducers.js:51:25)
      at combineReducers (node_modules/redux/lib/combineReducers.js:107:5)
      at Object.<anonymous> (src/data/reducers/index.js:14:122)
      at Object.<anonymous> (src/data/store.js:5:43)
      at Object.<anonymous> (src/data/store.test.jsx:1:116)
          at Generator.next (<anonymous>)
          at Promise (<anonymous>)
          at Generator.next (<anonymous>)
          at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:188:7)`

Need to investigate the best way to reach that global.

FYI @dsjen

Reply:
My guess is that the `let courseContext` is scoping it to this document only. What if you changed it to a `var` instead?

Also, there might be ways to mock it?